client/src/app/site/motions/modules/motion-list/components/motion-list/motion-list.component.ts

```diff
@@ -396,6 +404,67 @@ export class MotionListComponent extends ListViewBaseComponent<ViewMotion, Motio
         };
     }
 
+    /**
+     * Function to get a list with all favorite motions.
+     *
+     * @returns An array containing all favorite motions.
+     */
+    public motionsAsFavorite(): ViewMotion[] {
```

Comment:
Please rebase on the current master. Then there is a better way to get the amount of favorite motions. For instances with a lot of motions, this functions might be slow, becuase with every view update by angular.